src/components/SecurityRequirement/OAuthFlow.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { OpenAPISecurityScheme } from '../../types';
+import { SecurityRow } from './styled.elements';
+import { SeeMore } from '../SeeMore/SeeMore';
+import { Markdown } from '../Markdown/Markdown';
+
+export interface OAuthFlowProps {
+  type: string;
+  flow: OpenAPISecurityScheme['flows'][keyof OpenAPISecurityScheme['flows']];
+  RequiredScopes?: JSX.Element;
+}
+
+export class OAuthFlow extends React.PureComponent<OAuthFlowProps> {
```

Comment:
It would be great to use functional component here

Reply:
done

Reply:
@anastasiia-developer  Previously it was a `PureComponent` after refactoring it to a function component we need to keep the performance in the rendering perspective. So we might use React `memo`  HOC here, what do you think?

Reply:
yes, we might
I'm going to add that

Reply:
Done